### ISSUE TITLE
Empty matrix fails UtEqualToMatcher

### ISSUE THREAD

**Issue by VDFaller** [sassoftware/jsl-hamcrest#82]
```jsl
case = uttestcase("Test");
uttest(case, "empty matrix", 
	Expr(
		x = [];
		utassertthat(Expr(x), [])
	)
);
```
returns 
```jsl
FAILURE  x equal to [] (Test ⮚ empty matrix ⮚ 1)
    Actual: threw "Error: invalid subscript (must be number or list of numbers). 1 time. actual[ /*###*/Loc(!actual missing)]"
    Expected: equal to []
```

**Comment by emccorkle** [comment]
I think this may be fixed in master but I can't confirm right now. See #72 and #73.

**Comment by JustinChilton** [comment]
Yep, this is the issue that was fixed in #73.
